ci/test.sh

```diff
@@ -21,6 +21,19 @@ activate_local_venv
 
 git submodule update --init
 
+GCLOUD=`which gcloud`
+GCP_BIN=`dirname $GCLOUD`
+SDK_ROOT=`dirname $GCP_BIN`
+echo "Cloud SDK Root: $SDK_ROOT"
+$GCLOUD components install app-engine-python app-engine-python-extras cloud-datastore-emulator
+
+GCP_SDK_ROOT=$SDK_ROOT/platform/google_appengine
+GCP_SDK_YAML=`find $GCP_SDK_ROOT -type d | grep "yaml-.*[0-9]$"`
```

Comment:
I'm lost, what is the Cloud SDK yaml and why are we adding it to the path?

Reply:
Blame CircleCI and Google.  Importing config caused an ndb import which then caused a yaml import. For whatever reason the Google SDK couldn't import the yaml library in the SDK without me adding it to the path.